fcrepo-configs/src/main/java/org/fcrepo/config/FedoraPropsConfig.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.fcrepo.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * General Fedora properties
+ */
```

Comment:
Please add `@author` javadoc... and ideally, `@since 2020-06-11`.
Our checkstyle used to enforce the `@author`... not sure why that is no longer working, hmm.

Reply:
Also, I learned that `@since` is supposed to list the first version the class was added so you can tell when it became part of the codebase. I used to list date, but now am trying to stick `@since 6.0.0` in new ones. 🤷

Reply:
done